frontend/src/store/objectStore.ts

```diff
@@ -235,6 +235,18 @@ export const useObjectStore = defineStore('objectStore', () => {
     );
   }
 
+  async function togglePublic(objectId: string, isPublic: boolean) {
+    try {
+      loading.value = true;
+      await objectService.togglePublic(objectId, isPublic);
+    } catch (error) {
+      console.error(`Toggle public: ${error}`); // eslint-disable-line no-console
```

Comment:
We should have this be something visual for the user above and beyond just a console notification.